java/integTest/src/test/java/glide/standalone/CommandTests.java

```diff
@@ -268,6 +271,17 @@ public void time() {
         assertTrue(Long.parseLong(result[1]) < 1000000);
     }
 
+    @Test
+    @SneakyThrows
+    public void save() {
+        String error = "Background save already in progress";
+        // use another client, because it could be blocked
```

Comment:
Are you closing the clients if you are not using default?

Reply:
`try` block closes the temporary client